sdks/java/core/pom.xml

```diff
@@ -732,6 +732,12 @@
       <version>1.0-rc2</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.google.auto.value</groupId>
+      <artifactId>auto-value</artifactId>
+      <version>1.1</version>
+      <scope>provided</scope>
```

Comment:
+1 it should be compile-time only, aka `<scope>compile</scope><optional>true</optional>`